sql/core/src/test/scala/org/apache/spark/sql/execution/SparkPlanTest.scala

```diff
@@ -98,7 +100,7 @@ class SparkPlanTest extends SparkFunSuite {
       planFunction: Seq[SparkPlan] => SparkPlan,
       expectedAnswer: Seq[Row],
       sortAnswers: Boolean = true): Unit = {
-    SparkPlanTest.checkAnswer(input, planFunction, expectedAnswer, sortAnswers) match {
+    SparkPlanTest.checkAnswer(input, planFunction, expectedAnswer, sortAnswers, sqlContext) match {
```

Comment:
This is minor, and does not have to be addressed in this PR, but why did we make that an `object`?  It seems like this just complicates function calls since now we can't access fields like the `sqlContext`.